DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/csr-pipes/Readme.md

```diff
@@ -0,0 +1,187 @@
+# CSR-Pipe
+This example demonstrate simple vector addition kernel that write to the Control Status Register (CSR) using host pipe interface (CSR-Pipe)
+
+## Invocation Interface
+By default, an un-decorated oneAPI kernel will have all its control signals and arguments passed through an MM-Agent interface.
```

Comment:
```suggestion
By default, an un-decorated oneAPI kernel will have all its control signals and arguments mapped into the IP component's control/status register (CSR).
```

Also, please include a figure from here (you may need to customize... let me know if you can't find one in here
https://intel-my.sharepoint.com/personal/paul_white_intel_com/Documents/Documentation/HLS%20Docs/kernel-top-levels.vsdx?web=1